mod.sp

```diff
@@ -64,10 +64,10 @@ mod "microsoft365_compliance" {
 
   require {
     plugin "azuread" {
-      version = "0.10.0"
+      min_version = "0.10.0"
     }
     plugin "microsoft365" {
-      version = "0.0.1"
+      min_version = "0.0.1"
```

Comment:
@khushboo9024 we can update this version to latest [v0.4.1](https://github.com/turbot/steampipe-plugin-microsoft365/blob/main/CHANGELOG.md#v041-2023-10-04)